Badge.md

```diff
@@ -1,3 +1,94 @@
 (Version >= 1.16.0)
 
-Please read: https://github.com/louislam/uptime-kuma/pull/1119
\ No newline at end of file
+- [Description](#description)
+  - [Status badge](#status-badge)
+  - [Ping, uptime and avg-response badge](#ping-uptime-and-avg-response-badge)
+  - [Certificate Expire badge](#certificate-expire-badge)
+  - [Response badge](#response-badge)
+  - [Badge Styles](#badge-styles)
+  - [Open Badge Generator](#open-badge-generator)
+- [Screenshots](#screenshots)
+
+
+# Description
+
+Status, ping, uptime, avg-response badges, cert-exp and response generated in node-js, no internet access required (based on [badge-maker](https://www.npmjs.com/package/badge-maker) aka. [shields.io](http://shields.io/))
+
+There are six API endpoints that return an SVG graphic for a given (public) monitor:
```

Comment:
Nit: Lets avoid adding a number that we need to update here ^^
(if we add another badge in the future and don't update this number, people will think we are morons that can't count to 7...)

```suggestion
These are the endpoints that return an SVG graphic for a given (public) monitor:
```